### ISSUE TITLE
is it possible to use multi weight files to vote

### ISSUE THREAD

**Issue by l1t1** [leela-zero/leela-zero#334]
for example,
wf1 vs(wf2+wf3+wf4)
for one move of wf1, imput it to wf2~4 respectively
then compare the output
if two of them choose same move that diff from the other, then use the Most one.

**Comment by isty2e** [comment]
No point in doing this.

**Comment by grolich** [comment]
Just a clarification on why there's no point for doing this in this project:

You're talking about a quorum - which was indeed used in similar games before (like the Akara engine for Shogi a few years  ago). The problem is that a quorum assumes you have several strong players which are trying to "convince" each other (in Akara, engines were retrying the quorum suggested move again if they haven't before) of good moves.

In Neural Network training, first of all. we are trying to find better weights to improve our agent, 
the whole point is "learning" - that is - we want to improve an agent, not maximize the use of an already "trained" one. 
For that we need to know what the current weights for each network and the exact move they cause it to suggest (or the change might be effectively random with respect to the loss function used). Knowing what other networks suggested is not much help in that case.

Second, as the current models aren't even close to being even close to decent (actually still very bad), majority ruling might have no useful statistical advantage at all.

But the main idea is the first, it doesn't help training the network at all. While it is very likely to do the opposite.

**Comment by sethtroisi** [comment]
SWA does a little bit of this.
No progress or discussion for a year, closing.
